cmd/juno/juno.go

```diff
@@ -91,6 +92,13 @@ func main() {
 		if err != nil {
 			return err
 		}
+
+		yamlConfig, err := yaml.Marshal(config)
+		if err != nil {
+			return err
+		}
+		fmt.Printf("Running Juno with Config:\n%s\n\n", string(yamlConfig))
+
 		fmt.Printf("Running Juno with Config:\n%s\n\n", string(configJSON))
```

Comment:
Instead of printing it separately, you should replace the json here.

Reply:
> Instead of printing it separately, you should replace the json here.

I Change it.
Is that ok?

Reply:
Yeap, please run `make format` as well.

Reply:
I get;

![Screenshot from 2023-08-10 11-21-34](https://github.com/NethermindEth/juno/assets/10325815/d6d61b9a-90c9-4913-b47a-64333689825a)

@IronGauntlets @joshklop please take a look if that is what you had in mind.

Reply:
Looks very clean. However, before merging we should aim to make the printed config copy-pastable. That means using the mapstructure keys when unmarshaling (e.g., `db-path` instead of `databasepath`). Otherwise it could be misleading and frustrating for users.